### PR TITLE
Update Dependencies - Part 1

### PR DESCRIPTION
**Pull request by SyntaxNode**
**github.com/bradfitz/gomemcache**
This was previously replaced with a Google fork via a Go Module redirect directive. I've switched this to a proper code replace and updated to the newest version.

**github.com/didip/tollbooth/v6**
Updated to the latest version, which is now using Go Modules so we don't need to declare the time package indirect link. The changes seem minimal / bug fixes. We no longer recommend use of this feature, so we expect most hosts to have it disabled by now. I believe the overall risk is minimal.

**github.com/gofrs/uuid**
**github.com/julienschmidt/httprouter**
**github.com/rs/cors**
**github.com/stretchr/testify**
Updated to the same latest versions we're using in Prebid Server. We've had no issues in that project so the risk seems minimal.

### CODE REVIEW

File: validate.sh [hunk 1]
```diff
@@ -30,3 +30,9 @@ else
 fi
 
 go test $(go list ./... | grep -v /vendor/)
+
+if $VET; then
```
Comment (guscarreon) [inline validate.sh:34]: Before this check, `VET` is not being set above anywhere. Should we initialize in a Prebid Server fashion?
```
 1   #!/bin/bash
 2  
 3   set -e
 4  
 5   AUTOFMT=true
   + VET=true
   +
 6  
 7   while true; do
 8     case "$1" in
 9        --nofmt ) AUTOFMT=false; shift ;;
   +      --novet ) VET=false; shift ;;
10        * ) break ;;
11     esac
12   done
13  
14   die() { echo -e "$@" 1>&2 ; exit 1;  }
15   *-- 18 lines: check there are no formatting issues-------------------------
33  
34   if $VET; then
35     COMMAND="go vet"
36     echo "Running: $COMMAND"
37     `$COMMAND`
38   fi
```

File: backends/aerospike_test.go [hunk 2]
```diff
@@ -253,7 +253,7 @@ func TestClientPut(t *testing.T) {
 			desc: "AerospikeBackend.Put() does not throw error",
 			inAerospikeClient: &goodAerospikeClient{
 				records: map[string]*as.Record{
-					"defaultKey": &as.Record{
+					"defaultKey": {
```
Comment (guscarreon) [inline backends/aerospike_test.go:256]: Makes perfect sense to remove `&as.Record`. Good catch

Comment (SyntaxNode) [reply backends/aerospike_test.go:256]: That came from adding the simplification flag to gofmt, as we do for PBS. Does make sense though, I agree.

**Review by guscarreon — APPROVED** [review]
LGTM

**Review by bsardo — APPROVED** [review]
LGTM

**Pull request merged**